Book-A-Desk.Api/BookingNotifier.fs

```diff
@@ -40,7 +40,7 @@ module rec BookingNotifier =
         let sendOfficeRestrictionNotification (booking: Booking) = asyncResult {
                 let config = getEmailServiceConfiguration()
                 let! office = getOffice booking.Office.Id getOffices
-                let mailText = createOfficeRestriction office
```

Comment:
Please, rollabck this change as we need to add the proper message and it can be multi-line.